Iterable-iOS-SDK/IterableDeeplinkManager.m

```diff
@@ -53,6 +62,10 @@ -(void)getAndTrackDeeplink:(NSURL *)webpageURL callbackBlock:(ITEActionBlock)cal
     } else {
         NSURLSessionDataTask *trackAndRedirectTask = [redirectUrlSession
                                                       dataTaskWithURL:webpageURL completionHandler:^(NSData *data, NSURLResponse *response, NSError *error) {
+                                                          // Store attribution information
+                                                          if (deepLinkCampaignId) {
```

Comment:
You create the attributionInfo but I don't see a call to saveValueToUserDefaults to store the new expiration.
Perhaps you should refactor savePushPayload so that deeplinks also update the attribution window.

Reply:
attributionInfo is a property. In the setter you will see that it stores it to UserDefaults.